readthedocs/oauth/management/commands/load_project_remote_repo_relation.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from django.core.management.base import BaseCommand
+
+from readthedocs.oauth.models import RemoteRepository
+
+
+class Command(BaseCommand):
+    help = "Load Project and RemoteRepository Relationship from JSON file"
+
+    def add_arguments(self, parser):
+        # File path of the json file containing relationship data
+        parser.add_argument(
+            '--file',
+            required=True,
+            nargs=1,
+            type=str,
+            help='File path of the json file containing relationship data.',
+        )
+
+    def handle(self, *args, **options):
+        file = options.get('file')[0]
+
+        try:
+            # Load data from the json file
+            with open(file, 'r') as f:
+                data = json.load(f)
+        except Exception as e:
+            self.stdout.write(
+                self.style.ERROR(
+                    f'Exception occurred while trying to load the file "{file}". '
+                    f'Exception: {e}.'
+                )
+            )
+            return
+
+        for item in data:
+            try:
+                RemoteRepository.objects.filter(
+                    remote_id=item['remote_id']
+                ).update(project_id=item['project_id'])
```

Comment:
I think it will be good to log if the updated wasn't done for any reason. Here, if `remote_id` is not found, we won't notice it and we will think that everything worked correctly, even if there is 0 updates.

So, we should check if there was any update and if not, log that the item was not updated.